### PR TITLE
Ecml migration

### PR DESCRIPTION
**Pull request by rheafernandes**

### CODE REVIEW

File: platform-jobs/samza/collection-migration/src/main/java/org/ekstep/jobs/samza/service/CollectionMigrationService.java [hunk 1]
```diff
@@ -28,324 +31,428 @@
...
+            //Checks if the content can be migrated or already has been
+            if (migrationService.checkIfValidMigrationRequest(ecmlNode)) {
+                String contentBody = util.getContentBody(contentId);
+                String imageContentBody;
+                if (RequestValidatorUtil.isEmptyOrNull(contentBody))
+                    throw new ClientException(migrationService.ECML_MIGRATION_FAILED, "Ecml body cannot be null");
+                //Add media maps of all contents (Both node and image node)
+                List<Map<String, Object>> mediaContents = migrationService.getMediaContents(contentBody);
+                if (isImage && migrationService.checkIfValidMigrationRequest(ecmlImageNode)) {
+                    imageContentBody = util.getContentBody(contentId + ".img");
```
Comment (amitpriyadarshi) [inline platform-jobs/samza/collection-migration/src/main/java/org/ekstep/jobs/samza/service/CollectionMigrationService.java]: Add null check condition

File: platform-jobs/samza/collection-migration/src/main/java/org/ekstep/jobs/samza/service/CollectionMigrationService.java [hunk 2]
```diff
@@ -28,324 +31,428 @@
...
+                    String artifactUrl = migrationService.uploadAsset((File) fileMap.get(driveUrl).get(0), id);
+                    if (StringUtils.isNotBlank(artifactUrl))
+                        driveArtifactMap.put(driveUrl, artifactUrl);
+                });
+                //Delete all the downloaded content
+                migrationService.deleteDownloadedContent(downloadFileLocation);
+                //Update the ecml body and the nodes
+                if (MapUtils.isNotEmpty(driveArtifactMap)) {
+                    if (isImage)
+                        migrationService.ecmlBodyUpdate(contentBody, contentId + ".img", driveArtifactMap);
```
Comment (amitpriyadarshi) [inline platform-jobs/samza/collection-migration/src/main/java/org/ekstep/jobs/samza/service/CollectionMigrationService.java]: contentBody should be replaced with imageContentBody

Comment (rheafernandes) [reply platform-jobs/samza/collection-migration/src/main/java/org/ekstep/jobs/samza/service/CollectionMigrationService.java]: Done Lead Sir

File: platform-jobs/samza/collection-migration/src/main/java/org/ekstep/jobs/samza/service/CollectionMigrationService.java (hunk 1 above)
Comment (rheafernandes) [reply platform-jobs/samza/collection-migration/src/main/java/org/ekstep/jobs/samza/service/CollectionMigrationService.java]: Done Lead Sir

**Pull request merged**